src/components/include/utils/threads/thread.h

```diff
@@ -83,6 +83,25 @@ Thread* CreateThread(const char* name, ThreadDelegate* delegate);
 void DeleteThread(Thread* thread);
 
 class Thread {
+ public:
+  /**
+   * @brief ThreadState informs outside world about its state
+   * kThreadStateNone - there is no thread at all
+   * kThreadStateCreated - the thread is in state created (pthread_create has
+   * been called and it returned success)
+   * kThreadStateRunning - thread is in state running (executing
+   * delegates threadMain())
+   * kThreadStateStop - the thread is in state stop
+   * kThreadStateFinalize - the thread is in state finalize
+   */
+  enum ThreadState {
+    kThreadStateNone,
+    kThreadStateCreated,
+    kThreadStateRunning,
+    kThreadStateStop = 4,
+    kThreadStateFinalize = 8
```

Comment:
Are these names valid by a code style? I would change them to 
`NONE,
CREATED,
RUNNING,
STOP,
FINALIZE`

Reply:
@YarikMamykin 
Yes, these name are valid by a code style.
Your proposed option is invalid.
And additionally, it is not clear who CREATED or RUNNING thread or delegate ?
I would suggest leaving unchanged